README.md

```diff
@@ -54,6 +54,96 @@ Model::search('query')
 ```
 
 
+## `with`
```

Comment:
To be clear on usage, there are plenty of situations you need access to algolia search results in back end code. For example we are using algolia to pull in the nearest accommodations based around a lat/lon and rating and need hydrated models so we can then call methods on those results like generateUrl() or $accommodation->getLocation() etc.